manifests/cli_helper.pp

```diff
@@ -2,9 +2,7 @@
 #
 # A helper script for creating resources via the Jenkins cli
 #
-class jenkins::cli_helper (
-  $ssh_keyfile = undef,
```

Comment:
My preference would be that this argument stays so it's not used, then adding a deprecation warning if it's being used will ensure that we don't break current implementations